napari/layers/_base_layer/model.py

```diff
@@ -92,6 +95,39 @@ def name(self, name):
         self._name = name
         self.events.name()
 
+    @property
+    def indices(self):
+        """Tuple of slice objects for slicing arrays on each dimension."""
+        return self._indices
+
+    @indices.setter
+    def indices(self, indices):
+        if indices == self.indices:
+            return
+        self._indices = indices[-self.ndim:]
+        self._set_view_slice()
+
+    @property
+    def cursor_position(self):
+        """tuple: 2-tuple of cursor in image coordinates. Setter does
+        transformation from canvas coordinates to image coordinates
+        """
+        return self._cursor_position
+
+    @cursor_position.setter
+    def cursor_position(self, cursor_position):
+        transform = self._node.canvas.scene.node_transform(self._node)
+        self._cursor_position = tuple(transform.map(cursor_position)[:2])
```

Comment:
Can you add a note about the 2D-only nature of this? I'm puzzled about why they are leading rather than lagging dimensions here.

Reply:
as mentioned in the other comment, i'm eliminating this property as it was more confusing than useful